content/en/serverless/installation/go.md

```diff
@@ -78,8 +74,135 @@ datadog-ci lambda instrument -f my-function -f another-function -r us-east-1 -e
 
 More information and additional parameters can be found in the [Datadog Serverless CLI][2].
 
+### Set up tracing
+1. Set the environment variable `DD_API_KEY` to your Datadog API key from [API Management][3].
+2. Set the environment variable `DD_TRACE_ENABLED` to `true`.
```

Comment:
Also worth checking if Datadog CI already does this. Either way, we should also probably clarify that we are talking here about setting the environment variable for the functions, not in the development environment.